src/screens/LoginScreen/index.tsx

```diff
@@ -84,7 +84,11 @@ const LoginScreen: FC = () => {
             <Image mt={5} alt="CMI logo" size="sm" source={whiteLogo} />
           </Center>
 
-          {/*<Pressable onPress={() => {}}>/!*<Icon as={FontAwesome} name="database" />*!/</Pressable>*/}
+          <Flex mr={4} mb={6} alignItems="flex-end">
+            <Pressable onPress={navigateToAppLanguage}>
```

Comment:
We might not need to use Pressable here as the Icon component comes with a built-in Icon.Button component which is pressable.
https://github.com/oblador/react-native-vector-icons#iconbutton-component